src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +33,25 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        LocalDate localDate = LocalDate.now();
+        String result = "";
```

Comment:
do we need this local variable? let's return value at all case block

Reply:
done